keyboards/crkbd/rev2/info.json

```diff
@@ -0,0 +1,233 @@
+{
+    "usb": {
+        "pid": "0x0002"
+    },
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "lto": true,
+        "mousekey": true,
+        "nkro": true,
+        "encoder": true,
+        "oled": true,
+        "rgblight": false,
+        "rgb_matrix": true
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "direct":[
+            ["GP22", "GP20", "GP23", "GP26", "GP29",  "GP0",  "GP4"],
+            ["GP19", "GP18", "GP24", "GP27",  "GP1",  "GP2",  "GP8"],
+            ["GP17", "GP16", "GP25", "GP28",  "GP3",  "GP9",   null],
+            [  null,   null,   null, "GP14", "GP15", "GP11",   null]
+        ]
+    },
+    "encoder": {
+        "rotary": [
+            {"pin_a": "GP5", "pin_b": "GP7"},
+            {"pin_a": "GP6", "pin_b": "GP7"}
+        ]
+    },
+    "split": {
+        "enabled": true,
+        "matrix_pins": {
+            "right": {
+                "direct": [
+                    [ "GP8",  "GP9",  "GP3",  "GP2",  "GP1", "GP27", "GP25"],
+                    ["GP11", "GP14",  "GP4",  "GP0", "GP28", "GP26", "GP23"],
+                    ["GP15", "GP18",  "GP5", "GP29", "GP20", "GP22",   null],
+                    [  null,   null,   null, "GP16", "GP17", "GP19",   null]
+                ]
+            }
+        },
+        "encoder": {
+            "right": {
+                "rotary": [
+                    {"pin_a": "GP24", "pin_b":  "GP7"},
+                    {"pin_a":  "GP6", "pin_b":  "GP7"}
+                ]
+            }
+        }
+    },
+    "layouts": {
+        "LAYOUT_split_3x6_3_ex2": {
```

Comment:
Consider adding `LAYOUT_split_3x5_3` and `LAYOUT_split_3x6_3` here, so that the keymaps made for the previous versions could also work on this version without changes.  This would also be useful for people who want to use OLEDs instead of extra keys on the new version.

Reply:
Enable `LAYOUT_split_3x5_3` and `LAYOUT_split_3x6_3` at rev2